app/views/patient_mailer/closed_email.html.erb

```diff
@@ -1,15 +1,15 @@
 <% content_for :preheader do %>
-  <%= t('assessments.email.closed.header', locale: @lang || :en) %>
+  <%= t('assessments.email.closed.header', locale: @lang) %>
```

Comment:
is `@lang` guaranteed to not be `nil` anymore here?

Reply:
Yes. `patient.select_language` has `Languages.supported_language?(primary_language) ? primary_language&.to_sym : :eng`

Reply:
in the `PatientMailer` `@lang = patient.select_language`  and if you look at select_language it defaults to eng